packages/babel-compat-data/scripts/build-data.js

```diff
@@ -7,7 +7,7 @@ const flattenDeep = require("lodash/flattenDeep");
 const isEqual = require("lodash/isEqual");
 const mapValues = require("lodash/mapValues");
 const pickBy = require("lodash/pickBy");
-const unreleasedLabels = require("../data/unreleased-labels");
+const { unreleasedLabels } = require("../../babel-helper-compilation-targets");
```

Comment:
Can we just require `@babel/helper-compilation-targets`?

Reply:
Good catch; I'll update it. Note that this file isn't published to npm so it isn't a problem, but using `@babel/helper-compilation-targets` is more clear.